tests/MongoDBDriver_APM-001.phpt

```diff
@@ -0,0 +1,578 @@
+--TEST--
+HHVM-190: APM Specification
```

Comment:
This is a rather large test. I realize you need to insert some data before you issue a query, but I'd suggest moving the command failure to its own PHPT file to cut down on the expectation.